examples/documentation_examples/examples.go

```diff
@@ -2097,3 +2105,623 @@ func ChangeStreamExamples(t *testing.T, db *mongo.Database) {
 
 	atomic.StoreInt32(&stop, 1)
 }
+
+// AggregationExamples contains examples of aggregation operations.
+func AggregationExamples(t *testing.T, db *mongo.Database) {
+	ctx := context.Background()
+
+	salesColl := db.Collection("sales")
+	airlinesColl := db.Collection("airlines")
+	airAlliancesColl := db.Collection("air_alliances")
+
+	err := salesColl.Drop(ctx)
+	require.NoError(t, err)
+	err = airlinesColl.Drop(ctx)
+	require.NoError(t, err)
+	err = airAlliancesColl.Drop(ctx)
+	require.NoError(t, err)
+
+	{
+		date20180208 := parseDate(t, "2018-02-08T09:00:00.000Z")
+		date20180109 := parseDate(t, "2018-01-09T07:12:00.000Z")
+		date20180127 := parseDate(t, "2018-01-27T09:13:00.000Z")
+		date20180203 := parseDate(t, "2018-02-03T07:58:00.000Z")
+		date20180205 := parseDate(t, "2018-02-05T06:03:00.000Z")
+		date20180111 := parseDate(t, "2018-01-11T07:15:00.000Z")
+
+		// Start Aggregation Example 1
```

Comment:
Populating the collections isn't as an example, it and it means that all the examples are off by one, like 2 should be 1 and so on

Reply:
I included the populating as examples because many other functions in the file seem to do that as their first example (see `DeleteExamples` [here](https://github.com/mongodb/mongo-go-driver/blob/master/examples/documentation_examples/examples.go#L1633), or `QueryTopLevelFieldsExamples` [here](https://github.com/mongodb/mongo-go-driver/blob/master/examples/documentation_examples/examples.go#L148)). If it doesn't seem important to show the underlying collections' structures, then I can certainly remove them as examples.

Reply:
The listed examples should have the same numbers as the ones they're assigned in the ticket, so the docs team can scrape them later and put them in the right spots. For example, the delete documentation has a specific section about populating the collection that Example 55 in DeleteExamples maps to: https://docs.mongodb.com/manual/tutorial/remove-documents/index.html
They haven't asked for one for any of these, so I wouldn't include it.

Reply:
That makes sense! Thanks for looking at the other documentation. I removed the collection-population from the examples, and they shouldn't be off by one anymore.